scripts/pi-hole/js/queries.js

```diff
@@ -282,6 +282,7 @@ $(document).ready(function() {
             this.style.color = "";
           }
         );
+        $("td:eq(4)", row).unbind(); // Release any possible previous onClick event handlers
```

Comment:
`.unbind` is deprecated and has been superseded by the `.off()` method, more details on https://api.jquery.com/unbind.
```suggestion
        $("td:eq(4)", row).off(); // Release any possible previous onClick event handlers
```